compose/cli/docker_client.py

```diff
@@ -34,5 +34,5 @@ def docker_client():
             ca_cert=ca_cert,
         )
 
-    timeout = int(os.environ.get('DOCKER_CLIENT_TIMEOUT', 60))
+    timeout = int(os.environ.get('COMPOSE_HTTP_TIMEOUT', os.environ.get('DOCKER_CLIENT_TIMEOUT', 60)))
```

Comment:
We should deprecate `DOCKER_CLIENT_TIMEOUT`, i.e. show a deprecation warning if it's set, much as we do with the `FIG_*` variables: https://github.com/docker/compose/blob/17682c58db8986f1d1492559d72b3960b86f5f05/compose/cli/command.py#L92-L94